samples/JsInterop/index.js

```diff
@@ -0,0 +1,41 @@
+
+import { dotnet } from './dotnet.js'
+
+let dotnetExports = null
+
+async function createRuntimeAndGetExports() {
+    const { getAssemblyExports, getConfig } = await dotnet.create()
+    const config = getConfig()
+    return getAssemblyExports(config.mainAssemblyName)
+}
+
+export async function parseAsync(jsonTexts) {
+    /* Start dotnet runtime if not already started */
+    dotnetExports ??= await createRuntimeAndGetExports()
+
+    jsonTexts = typeof jsonTexts === 'string' ? [jsonTexts] : jsonTexts
+
+    let objectModelJson
+    try {
+        objectModelJson = await dotnetExports.ModelParserInterop.ParseToJsonAsync(Array.from(jsonTexts))
+    } catch (modelParsingError) {
+        try {
+            /**
+             * Error info was marshalled accross the interop boundary using the error message.
+             * Parse the error info JSON into a more usable form in JS.
+             */
+            modelParsingError.exceptionInfo = JSON.parse(error.message)
```

Comment:
Caught by @jrdouceur in the previous PR
```suggestion
            modelParsingError.exceptionInfo = JSON.parse(modelParsingError.message)
```